api/user/relations.py

```diff
@@ -0,0 +1,11 @@
+from api.app import db
+
+author_comments = db.Table(
+    db.Column("auth_id", db.Integer, db.ForeignKey("author.id")),
+    db.Column("comment_id", db.Integer, db.ForeignKey("comment.id")),
+)
```

Comment:
I'm unclear what this is doing - the docs seem to mention a many-to-many relationship - is that what we want here? author -> comments is one to many

Reply:
the name is confusing but this relation is for author_likes_comments. I guess that would be a better name